tests/test_core.py

```diff
@@ -6665,37 +6665,23 @@ def test_freetype(self):
   @no_asan('local count too large for VMs')
   @no_ubsan('local count too large for VMs')
   @is_slow_test
-  def test_sqlite(self):
+  @parameterized({
+    'single': (False,),
+    'pthreads': (True,),
+  })
+  def test_sqlite(self, use_pthreads):
+    if use_pthreads:
+      self.set_setting('USE_PTHREADS')
+      self.setup_node_pthreads()
     self.set_setting('EXPORTED_FUNCTIONS', ['_main', '_sqlite3_open', '_sqlite3_close', '_sqlite3_exec', '_sqlite3_free'])
     if '-g' in self.emcc_args:
       print("disabling inlining") # without registerize (which -g disables), we generate huge amounts of code
       self.set_setting('INLINING_LIMIT')
```

Comment:
I wonder if this block is still needed?  Can you try `core2g.test_sqlite` without this .. if it passed in a reasonable amount of time I think we should remove this block... since it no longer effects the compilation of sqlite itself.. only the tiny benchmark code.

Reply:
Is this a reasonable amount of time?

```
Using 2 parallel test processes
test_sqlite_single (test_core.core0) ... ok (1.23s)
test_sqlite_pthreads (test_core.core0) ... ok (1.39s)

DONE: combining results on main thread

test_sqlite_pthreads (test_core.core0) ... ok
test_sqlite_single (test_core.core0) ... ok

----------------------------------------------------------------------
Ran 2 tests in 1.602s
```